docs/http-api/src/topics/authentication.md

```diff
@@ -0,0 +1,45 @@
+# Authentication
+
+Qaul supports two forms of authentication for the HTTP Api. In the first a cookie
+is set on the client, which if present in future requests will authenticate the 
+client. The second provides the client with a token. The client then provides the token
+in it's `Authorization` header on future requests.
+
+## Obtaining a grant
+
+The first step of dealing with authenticated qaul endpoints is to obtain a grant.
+This is done through the [`login`](/endpoints/login.html) endpoint by sending a
+[`user_auth`](/entities/user_auth.html`) entity. Below is an example of the body of
+such a request:
+
+```
+{
+	"data": {
+		"id": "0000000000000000",
+		"type": "user_auth",
+		"attributes": {
+			"secret": "my super secret password",
+			"grant_type": "token",
+		}
+	}
+}
```

Comment:
This code block seems to be indented with a mix of tabs and spaces

Reply:
That's very strange as it's not on my local copy